draft_fifa/context_processors.py

```diff
@@ -1,7 +1,12 @@
+import uuid
+from typing import List
+
+from django.core.handlers.wsgi import WSGIRequest
+
 from fifa_draft.models import Group
 
 
-def pick_alert(request):
+def pick_alert(request: WSGIRequest) -> dict[str, bool | List[uuid.UUID]]:
```

Comment:
Gitara. Czyli jednak python 3.9? Bo na 3.8 to nie zadziała.

W przypadku 3.8 zamiast `bool | List[uuid.UUID]` byłoby `Union[bool, uuid.UUID]` (`from typing import Union`).

Reply:
Tu zmieniałem na wersję działającą z pythonem 3.8 po wysłaniu PR, bo po paru godzinach zauważyłem że na Heroku nie odpala się apka przez nierozpoznane typowanie.